### ISSUE TITLE
modules are listed as unauthorized when PAUSE indexed them

### ISSUE THREAD

**Issue by karenetheridge** [metacpan/metacpan-api#584]
ome modules are listed here as unauthorized, but PAUSE indexed them just fine -- https://metacpan.org/release/MooseX-Singleton (0.30 release).  When I made the release, PAUSE said it was happy with all the modules therein; I did no reindexing afterwards.  02packages.details.txt also lists all the modules correctly against the new release.  (There are some ancient modules still in the index under SARTAK that have been deleted from the dist a while back; they are not relevant.)

**Comment by mickeyn** [comment]
hi @karenetheridge 

I can't recreate the issue now, and I see all files for this release are authorized in the ES.
Can you please confirm?

On my VM with a slightly older 02packages/06perms I got the indexer to report some modules in this release as unauthorized, which I am assuming is what happend.
I can think of one race situation in which our watcher will catch a new file before the perms files gets updated (*if* they get updated in the same rsync)... but not sure if it applies here.

**Comment by karenetheridge** [comment]
Agreed - I force-refreshed the release/ page and now all modules show up as
authorized.  Sounds like it is indeed a race condition!
